charts/orchestrator/README.md

```diff
@@ -2,7 +2,7 @@
 Orchestrator
 ===========
 
-Helm chart to deploy the Orchestrator solution suite on OpenShift, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service.
+Helm chart to deploy the Orchestrator solution suite on OpenShift, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service. #magic___^_^___line
```

Comment:
same here, I think we can remove `Data Index and Job Service`

Reply:
this file is generated. I'll regenerate it to reflect the updated description from the Chart.yaml